api-specificatie/openapi.yaml

```diff
@@ -123,11 +157,19 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/VolgindicatiePersonen"
+                $ref: "#/components/schemas/VolgindicatiePersoon"
         '204':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/204"
+        '401':
```

Comment:
Ik denk dat ook een 400 response bij DELETE /volgindicatiepersonen/{burgerservicenummer} moet worden toegevoegd om dezelfde reden dat die nu is opgenomen bij GET /volgindicatiepersonen/{burgerservicenummer}

Reply:
Ik heb weer even gespiekt bij de API's voor zaakgericht werken. Daar komt een 400 in alle berichtsoorten voor behalve in een DELETE. Ik ben heel benieuwd wat daar precies de reden voor is. Wellicht heeft dat er ook hier mee te maken dat de OAS specs gegenereerd worden vanuit de server implementaties.

Reply:
Zie [hier ](https://github.com/VNG-Realisatie/gemma-zaken/issues/1661)voor meer informatie over het ontbreken van de 400 response bij de DELETE berichten in de API's voor zaakgericht werken.

Reply:
Het pad bevat hier een burgerservicenummer, die is gedefinieerd met een pattern, minLength en maxLength. 

/volgindicatiepersonen/12345678 zou volgens mij een 400 moeten geven want voldoet niet aan de minLength eis. De client app kan dan specifiek melden dat er een cijfer te weinig is ingevoerd.

/volgindicatiepersonen/123456789 zou volgens mij een 404 moeten geven want voldoet wel aan validatieregels, maar bestaat niet.